src/i9n/fungible.rs

```diff
@@ -215,13 +217,44 @@ impl Runtime {
         }
     }
 
-    pub fn sync(
+    pub fn export_asset(
+        &mut self,
+        asset_id: ContractId,
+    ) -> Result<Genesis, Error> {
+        match &*self.command(Request::ExportAsset(asset_id))? {
+            Reply::Failure(failure) => Err(Error::Reply(failure.clone())),
+            Reply::Genesis(response) => {
+                println!("Export asset succeeded");
```

Comment:
We must not use prints, we have logging crate for any outputs! And here even logging is not required since it is logged anyway when the request is sent

Reply:
Sorry, I found that you just used patter already present in other methods call which happens that it passed my review. Anyway, we need to get rid of all prints and use logging methods

Reply:
yep, I thought they weren't correct, but preferred to use the same pattern used in other already defined methods

Reply:
Ok, I see, the invoice deserialization must be done differently in SDK, so we do not need this here.